### PR TITLE
Replace the check boot version error with a nice message

### PR DESCRIPTION
**Pull request by Aireil**

### CODE REVIEW

File: XIVLauncher/Windows/MainWindow.xaml.cs [hunk 1]
```diff
@@ -251,7 +251,22 @@ private void HandleBootCheck(Action whenFinishAction)
...
+                try
+                {
+                    bootPatches = _launcher.CheckBootVersion(App.Settings.GamePath);
+                }
+                catch (Exception ex)
+                {
+                    Log.Error(ex, "Unable to check boot version.");
+                    MessageBox.Show(Loc.Localize("CheckBootVersionError", "XIVLauncher was not able to check the boot version for the select game installation. This can happen if a maintenance is currently in progress or if your connection to the version check server is not available. Please report this error if you are able to login with the official launcher, but not XIVLauncher."), "XIVLauncher", MessageBoxButton.OK, MessageBoxImage.Error);
+
+                    _ = Task.Run(SetupHeadlines);
```
Comment (goaaats) [inline XIVLauncher/Windows/MainWindow.xaml.cs:264]: Why the discard?

File: XIVLauncher/Windows/MainWindow.xaml.cs [hunk 2]
```diff
@@ -251,7 +251,22 @@ private void HandleBootCheck(Action whenFinishAction)
...
+                    Show();
+                    Activate();
+                    return;
```
Comment (goaaats) [inline XIVLauncher/Windows/MainWindow.xaml.cs:267]: You also need to set isLoggingIn = false here, bit messy, should redo error handling in MainWindow some time

**Review by goaaats — CHANGES_REQUESTED** [review]

File: XIVLauncher/Windows/MainWindow.xaml.cs (hunk 2 above)
Comment (Aireil) [reply XIVLauncher/Windows/MainWindow.xaml.cs:267]: Oops, yep, ctrl z more than I thought. In my opinion, it shouldn't need to Show() and Activate(), just return normally and handle it there. I just copied the maintenance error as this is the same thing.

File: XIVLauncher/Windows/MainWindow.xaml.cs (hunk 1 above)
Comment (Aireil) [reply XIVLauncher/Windows/MainWindow.xaml.cs:264]: It was just to suppress the warnings in the Task, not needed there though.

**Review by goaaats — APPROVED** [review]

**Pull request merged**